### DISCUSSION TITLE
2009/11/30/%E5%A6%82%E4%BD%95%E6%BF%80%E6%B4%BBlabview-scripting%E9%AB%98%E7%BA%A7%E5%8A%9F%E8%83%BD

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#286] [Announcements]
# 2009/11/30/%E5%A6%82%E4%BD%95%E6%BF%80%E6%B4%BBlabview-scripting%E9%AB%98%E7%BA%A7%E5%8A%9F%E8%83%BD

LabVIEW Scripting的高级功能（比如创建新的VI，修改VI程序框图上的程序等）是需要特殊授权的。从2009年8月开始，NI公司将这一授权公开给了所有LabVIEW用户。也就是任何一个LabVIEW用户都可以免费申请获得LabVIEW Scripting的高级功能的使用授权。

https://blog.qizhen.xyz/2009/11/30/%E5%A6%82%E4%BD%95%E6%BF%80%E6%B4%BBlabview-scripting%E9%AB%98%E7%BA%A7%E5%8A%9F%E8%83%BD

**Comment by ruanqizhen** [comment]
之前博客下的留言：

建军
2010年01月25日 11:34 编辑
感谢LVC，我为这事昨晚折腾了一晚上也没搞定，谢谢了

回复
LVC
2010年01月22日 12:08 编辑
在安装这个工具包的时候会出现一点问题，如果你的LabVIEW安装不是默认的路径，就是C：\\program file。。。如果不是以上路径，那么你的license manage无法发现已经安装的工具包具体解决方法参看下Ahh… I think I\’ve got it. The Scripting Installer doesn\’t detect if you installed the general NI software to a custom location (it finds LabVIEW, but not the general NI folder). Namely, it doesn\’t work unless the License Manager is installed to Program Files\\National Instruments\\Shared\\License Manager. I will try and correct the installer. Until then, after installing you can copyProgram Files\\National Instruments\\Shared\\License Manager\\Licenses\\LabVIEW_Scripting_PKG_080600.licto whatever location you have the License Manager installed to. Let me know if this works for you.

回复
LVC
2010年01月21日 13:33 编辑
非常感谢！！！如果不是阮大侠，根本不知道有这事，以前一直希望能够免费使用scripting功能！！！thanks，希望阮大侠以后能经常更新这方面的资料。

回复
